x-pack/test/security_solution_api_integration/package.json

```diff
@@ -7,404 +7,338 @@
   "scripts": {
     "initialize-server:genai:trial_complete": "node ./scripts/index.js server genai trial_license_complete_tier",
     "run-tests:genai:trial_complete": "node ./scripts/index.js runner genai trial_license_complete_tier",
-
```

Comment:
nit: unnecessary changes

Reply:
Oh, that is my fault. My editor formats 'json' files automatically.